mne/beamformer/_lcmv.py

```diff
@@ -166,15 +231,16 @@ def _apply_lcmv(data, info, tmin, forward, noise_cov, data_cov, reg,
         # SSP and whitening
         if info['projs']:
             M = np.dot(proj, M)
-        M = np.dot(whitener, M)
+        if noise_cov is not None:
+            M = np.dot(whitener, M)
 
         # project to source space using beamformer weights
-
         if is_free_ori:
             sol = np.dot(W, M)
             logger.info('combining the current components...')
             sol = combine_xyz(sol)
-            sol *= noise_norm_inv[:, None]
+            if weight_norm is not None:
```

Comment:
can you add here:

assert weight_norm == 'unit-norm-gain'

but I still have the feeling that this could be done when estimating the filters so this is not needed here.